testing/tests/index.test.js

```diff
@@ -78,3 +77,70 @@ test("content with non-ascii characters in the slug", () => {
   expect(doc.title).toBe("Bézier curve");
   expect(doc.mdn_url).toBe("/en-US/docs/Glossary/Bézier_curve");
 });
+
+test("content built bar page", () => {
+  expect(fs.existsSync(buildRoot)).toBeTruthy();
+
+  const builtFolder = path.join(buildRoot, "en-us", "docs", "web", "bar");
+  expect(fs.existsSync(builtFolder)).toBeTruthy();
+
+  const jsonFile = path.join(builtFolder, "index.json");
+  expect(fs.existsSync(jsonFile)).toBeTruthy();
+
+  // We should be able to read it and expect certain values
+  const { doc } = JSON.parse(fs.readFileSync(jsonFile));
+  expect(doc.title).toBe("bar: A collection of xref macro calls");
+  expect(doc.summary).toBe("This is the Bar test page.");
+  expect(doc.mdn_url).toBe("/en-US/docs/Web/Bar");
+  expect(doc.popularity).toBe(0.51);
+  expect(doc.modified).toBeTruthy();
+  expect(doc.source).toBeTruthy();
+
```

Comment:
I'm conflicted. Perhaps by talking about it it will get the brain juices going. 

Should there be a test here that inspects the `doc.flaws` object? Something that checks that we have the right bits of data available in there as it matters for future plans to *use* the errors to automate fixing things. 

But on the flip side, it does feel like an implementation detail. If we had a human QA person on our team, I'm not sure we would need to explain our requirements and expectations about the flaws objects.

Reply:
Actually, the more I think about it I agree with that last sentence. It *is* an implementation detail that is not important. What matters (at least here in this work on functional testing) is that the website works as expected. Not *how* it works.

Reply:
@peterbe You know, I'm thinking that perhaps I should check `doc.flaws` since it becomes part of the interface of the local server. In fact, when I was working on this PR, I explored accessing the flaws via the headless tests, but decided it was something to discuss with our team ("Do we want to also test the toolbar?").

Reply:
I’m one little comment away from fully reviewing this. But now distracted and on lunch break. I think, if we do test the flaws, it’ll be part of the trolling work that uses the flaws to auto-fix things.